waku/factory/external_config.nim

```diff
@@ -613,6 +614,22 @@ proc parseCmdArg*(T: type crypto.PrivateKey, p: string): T =
   except CatchableError:
     raise newException(ValueError, "Invalid private key")
 
+proc parseCmdArg*[T](_: type seq[T], s: string): seq[T] {.raises: [ValueError].} =
+  var
+    inputSeq: JsonNode
+    res: seq[T] = @[]
+
+  try:
+    inputSeq = s.parseJson()
+  except Exception:
+    raise newException(ValueError, "Could not parse sequence")
```

Comment:
Maybe useful to make it more explicit the sequence that couldn't be parsed
```suggestion
    raise newException(ValueError, fmt"Could not parse sequence [{s}]")
```

Reply:
Sounds goood, do we need to add the `[]`? Pretty sure it's already included in the string representing a seq

Reply:
> Sounds goood, do we need to add the `[]`? Pretty sure it's already included in the string representing a seq

In this particular case I think is interesting to add the `[]` so that it is explicit when `s` is empty.
Or, thinking it better, I think it is better to keep following our style of adding a colon (`:`) at the end